server/lib/validation/form_validators/ManageApplication.js

```diff
@@ -94,17 +92,23 @@ class ManageApplication extends Validator {
         this.errors.description = errorManifest.description.invalid;
       }
     }
-    for (const restrictedIp of restrictedIps){
-      if (restrictedIp.length > 0) {
-        if (!this.isValidIp(restrictedIp)) {
-          this.errors.restrictedIp = errorManifest.restrictedIp.invalid;
+    if (payload.restrictedIps.length > 0) {
```

Comment:
I'm surprised this handles nulls, I would have thought it still throws a null pointer when accessing length it payload doesn't have a value for.
Could we change that to `if !payload.restrictedIps { `
https://codereview.stackexchange.com/questions/5572/string-isnullorempty-in-javascript

Reply:
won't that send it into the block if restrictedIps is empty?

Reply:
Oh yeah, so drop the `!` would fix that right? My mistake.

Reply:
or `if payload.restrictedIps && payload.restrictedIps.length > 0 {`

Reply:
Ok, changed